src/OpenTelemetry/README.md

```diff
@@ -308,6 +305,21 @@ Core, Console.
 }
 ```
 
+Note: "Current directory" of your process is the working directory of your
```

Comment:
Consider putting a link to the official definition https://en.wikipedia.org/wiki/Working_directory.

Reply:
Working directory sounds like better well known name and unnecessary for adding extra explanation about it.

Reply:
I agree. After reading it again, I feel like the explanation is explaining the concept of the working directory rather than what the code is doing. Adding the wikipedia link might just do a better job in explaining the concept.